packages/core/utils.js

```diff
@@ -13,37 +13,29 @@ export const extractComponentFromChildren = (_children, componentName) => {
   return [children, component];
 };
 
-export const buildTheme = (components, base) => {
-  // Resolve references to base theme throughout component themes
-  // eg, `link.space.mx: 2` -> `link.space.mx: '1.6rem'`
-  // eg, `link.colors.hoverColor: 'primary'` -> `link.colors.hoverColor: '#738FFC'`
+export const buildTheme = (base, components) => {
```

Comment:
IIUC the only purpose of this function is to resolve the values inside component theme objects. What do you think of renaming the function to be more specific then? Eg `resolveComponentThemeValues()` or something like that.

Reply:
That definitely makes sense, but users are required to use it and I feel resolveComponentThemeValues is a little verbose. In the context of a custom theme I think buildTheme works pretty well for how the user uses it:

```
export default buildTheme(components, {
  colors: {
    // Base color overrides
    primary: "red",
    secondary: "#333"
  },
...
```